src/Core/Common/MultThreadOilPainting/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.10.2)
```

Comment:
The name of the example should be `MultiThreadedOilPainting` or `MultiThreadOilPainting`. Current name is `MultThread...` (missing **i**).

Reply:
```
vim $(git grep -l "MultThread")
```
Replaced MultThread with MultiThread